src/AdvancedNovaMediaLibraryServiceProvider.php

```diff
@@ -34,4 +37,28 @@ protected function routes()
             ->prefix('nova-vendor/ebess/advanced-nova-media-library')
             ->group(__DIR__.'/../routes/api.php');
     }
+
+    /**
+     * Load package translation resources.
+     *
+     * @return void
+     */
+    protected function loadTranslations()
+    {
+        $this->loadJSONTranslationsFrom(__DIR__ . '/../resources/lang');
+        $this->loadJSONTranslationsFrom(resource_path('lang/vendor/advanced-nova-media-library'));
+    }
+
+    /**
+     * Bootstraps current application locale translations to Nova.
+     *
+     * @return void
+     */
+    protected function bootTranslations()
+    {
+        $locale = $this->app->getLocale();
+
+        Nova::translations(__DIR__ . "/../resources/lang/{$locale}.json");
+        Nova::translations(resource_path("lang/vendor/advanced-nova-media-library/$locale.json"));
```

Comment:
Use `lang_path('vendor/advanced-nova-media-library')` instead of `resource_path('lang/vendor/advanced-nova-media-library')`, as users might have configured a different path for their translations.